library/std/src/os/uefi/ffi.rs

```diff
@@ -0,0 +1,73 @@
+use crate::ffi::{OsStr, OsString};
+use crate::sealed::Sealed;
+use crate::sys_common::ucs2;
+
+#[unstable(feature = "uefi_std", issue = "100499")]
+pub trait OsStrExt: Sealed {
```

Comment:
These traits need ACP as well. For example, I think this probably should mimic the API of `std::os::windows::OsStrExt` rather than having functions named `to_ucs2`/`to_ffi_string`.